main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      squirrel
+//  action 1:  hibernates
+squirrel.hibernate();
+//  action 2:  gathers nuts
+squirrel.gatherNuts();
+//  action 3:  runs away
+squirrel.runsAway();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      cup
+//  action 1:  abides
+cup.abide();
+//  action 2:  shatters
+cup.shatter();
+//  action 3:  conveys liquid
+cup.conveyLiquid();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      krill
+//  action 1:  drifts
+krill.drift();
+//  action 2:  provides nutrients
+krill.provideNutrients();
+//  action 3:  swim
+krill.swim();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.